client/libs/ui-libraries/src/lib/searchbar/searchbar.component.ts

```diff
@@ -37,23 +31,20 @@ export class SearchbarComponent implements OnDestroy, OnInit {
     this.sermonSpeakersFacade.loadSermonSpeakers();
   }
 
-  ngOnDestroy() {
-    this.destroy$.next();
-    this.destroy$.unsubscribe();
-  }
-
-  search() {
+  searchSermons() {
     this.sermonSpeakersFacade.searchSermons(this.form.value);
   }
 
-  selectCustom(searchType: string, index = 0) {
+  selectCustom({ searchType, index }) {
     const datepickerOption = 2;
 
-    this.form.patchValue({ searchType });
-
     if (index === datepickerOption) {
       this.patchSelectedDate();
+
+      return;
     }
+
```

Comment:
Also, unnecessary white-space

Reply:
Usually, we add a space when after an if statement to show clarity that something may happen after it.